README.md

```diff
@@ -21,6 +21,15 @@ docker run --rm -it -p 9229:9229 -p 8081:80 -e NODE_ENV=development local-form-m
 
 Running it in development mode and exposing port 9229 allows you to connect your debugger to the docker.
 
+### Project Structure
+
+The project follows the Model-view-controller (MVC) architecture.
+
+- Routing and anything Express related resides in `/controllers`. The routes should be simple methods that call services.
+- Services reside in `/services`. This is where the business logic and validations are.
+- Querries for the database or Comunica go in `/domain/data-access`. These query methods should only contain the means to query or mutate the data, and return the result in a format that the service understands.
```

Comment:
Looked up the comma before the and as it looked weird to me xd sorry

Totally fine ;p https://www.grammarly.com/blog/comma-before-and/